src/main/java/cart/product/controller/AdminController.java

```diff
@@ -0,0 +1,52 @@
+package cart.product.controller;
+
+import cart.product.dto.RequestProductDto;
+import cart.product.dto.ResponseProductDto;
+import cart.product.service.ProductListService;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/admin")
+public class AdminController {
+    
+    private final ProductListService productListService;
+    
+    public AdminController(final ProductListService productListService) {
+        this.productListService = productListService;
+    }
+    
+    @GetMapping("/product/list")
+    @ResponseStatus(HttpStatus.OK)
+    public List<ResponseProductDto> display() {
+        return this.productListService.display();
+    }
+    
+    @PostMapping("/product/create")
```

Comment:
uri를 통해 자원을 명시하고 http method로 crud를 표현하는게
좀더 restful에 가까울것 같아요 
```suggestion
    @PostMapping("/product")
```
여기선 product 만 있으면 될것 같아요